latexify/core.py

```diff
@@ -82,6 +82,26 @@ def visit_Set(self, node, action):  # pylint: disable=invalid-name
 
   def visit_Call(self, node, action):  # pylint: disable=invalid-name
     """Visit a call node."""
+
+    def _decorated_lstr_and_arg(node, callee_str, lstr):
+      """Decorate lstr and get its associated arguments"""
+      if callee_str == 'sum' and isinstance(node.args[0], ast.GeneratorExp):
```

Comment:
The more functions we support decorations for, the bigger `visit_Call' will be. Is acceptable or is there any better way to implement this type of decorations keeping future expandability in mind?

Reply:
We can separate some routines out of this method if needed. But it looks reasonable to keep the current implementation because we are supporting only `sum`; it is too early to generalize the behavior.